IPython/core/shellapp.py

```diff
@@ -100,7 +100,7 @@
 
 class InteractiveShellApp(Configurable):
     """A Mixin for applications that start InteractiveShell instances.
-    
+
```

Comment:
Sorry! I didn’t even notice Atom did that on save. If I contribute more in
the future I’ll fix that.

On Tue, Jul 26, 2016 at 4:24 PM Thomas Kluyver <notifications@github.com>
wrote:

> In IPython/core/shellapp.py
> <https://github.com/ipython/ipython/pull/9813#discussion_r72329425>:
>
> > @@ -100,7 +100,7 @@
> >
> >  class InteractiveShellApp(Configurable):
> >      """A Mixin for applications that start InteractiveShell instances.
> > -
> > +
>
> Not a big deal, but if you're contributing to IPython/Jupyter more in the
> future, we will ask you to configure your editor not to strip whitespace in
> lines you haven't touched - it can create needless merge conflicts, and it
> makes git blame less useful.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ipython/ipython/pull/9813/files/57522575ad2a69eaa7a931f98221b758d787fd74#r72329425>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AJ5Yr3y6_sBxQNFmFcDYKfSgPZLIlAF_ks5qZmzsgaJpZM4JVivP>
> .
>